ksqldb-rest-app/pom.xml

```diff
@@ -177,6 +177,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
+            <artifactId>system-rules</artifactId>
+            <version>1.19.0</version>
```

Comment:
nit: all dependencies should be declared in the parent pom and versions should be declared as properties